forms-flow-web/src/apiManager/services/bpmServices.js

```diff
@@ -73,3 +77,18 @@ export const getPaginatedForms = (forms, limit, page, sort) => {
   forms.sort(dynamicSort(sort));
   return forms.slice((page - 1) * limit, ((page - 1) * limit) + limit);
 };
+
+export const deployBpmnDiagram = (data, token, isBearer = true) => {
+
+  const url = BPM_BASE_URL + "/deployment/create";
+  
+  return axios.post(url, data, {
```

Comment:
Can update and use the existing Post (content Type we would need to update to dynamic), https://github.com/AOT-Technologies/forms-flow-ai/blob/2bcf8462c7edb486cbe4c9eead666cb1e92a8596/forms-flow-web/src/apiManager/httpRequestHandler/index.js#L18

Reply:
Done